dockers/docker-base-bullseye/etc/rsyslog.conf

```diff
@@ -0,0 +1,76 @@
+#
+#  /etc/rsyslog.conf    Configuration file for rsyslog.
```

Comment:
>/etc/rsyslog.conf [](http://example.com/codeflow?start=3&length=17)

This file is same as dockers/docker-base-stretch/etc/rsyslog.conf. Can we use a symlink in git repo?

Reply:
Also for other files such as .vimrc

Reply:
Git supports symlinks, but during the Docker build, the build context only covers files within the `dockers/docker-base-bullseye` folder, and since those files will now be outside of that folder, it fails there.